main.js

```diff
@@ -5,8 +5,17 @@ const bodyParser = require('body-parser');
 const http = require('http');
 const socketio = require('socket.io');
 const { instrument } = require('@socket.io/admin-ui');
+const mongoose = require('mongoose');
 const userRouter = require('./src/routes/user.route');
 
+mongoose
+  .connect(`mongodb://${process.env.MONGO_HOST}:27017/wizard?authSource=admin`, {
```

Comment:
Also add the `MONGO_HOST` to the README. We could also add some error handling if these variables are not set.